### PR TITLE
install runc r95

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -166,7 +166,7 @@ installMoby() {
 
 ensureRunc() {
     CURRENT_VERSION=$(runc --version | head -n1 | sed 's/runc version //' | sed 's/-/~/')
-    local TARGET_VERSION="1.0.0~rc92"
+    local TARGET_VERSION="1.0.0~rc95"
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:169]: do we not want to bake this to VHD first?

Comment (eriksywu) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:169]: this is also getting installed into the VHDs:

==> azure-arm: + ensureRunc
==> azure-arm: ++ runc --version
==> azure-arm: ++ head -n1
==> azure-arm: ++ sed 's/runc version //'
==> azure-arm: ++ sed s/-/~/
==> azure-arm: + CURRENT_VERSION=1.0.0~rc95
==> azure-arm: + local TARGET_VERSION=1.0.0~rc95
==> azure-arm: + '[' 1.0.0~rc95 '!=' 1.0.0~rc95 ']'

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:169]: I see. Let's finish all the test before putting it to VHD then.

**Comment by xuto2** [comment]
lgtm. pls make sure all e2e tests are run.

**Pull request merged**